value/const.go

```diff
@@ -81,6 +81,11 @@ func newF(conf *config.Config) *big.Float {
 	return new(big.Float).SetPrec(conf.FloatPrec())
 }
 
+// newFxP returns a new big.Float with extended precision = conf.FloatPrec() + prec.
```

Comment:
I would rename prec 'extra' throughout the CL. It's not the precision, it's the extra bits.

Reply:
That one was nagging me too. Done.